src/utils/tab.js

```diff
@@ -9,7 +9,10 @@ import WarningTextEnum from '../enums/WarningTextEnum';
 
 let tabUtils = {
   tabs: [],
-  config: {}
+  config: {
+    parse: JSON.parse,
```

Comment:
`postmessage.js` uses `tabUtils.config.parse`, so, why have you hardcoded this here? Shouldn't it come from the user-provided config otherwise files using `tabUtils.config.parse/stringify` would always rely on native methods?

Reply:
Chalk this up to safety coding ... I didn't know if `tabUtils` was ever used outside the context of the `Parent` class, so I was defaulting the values here (they would be overridden by instantiating the first `Parent class instance). If this is guaranteed, I can remove the defaulting.